las_geoh5/import_las.py

```diff
@@ -85,12 +86,14 @@ def get_depths(lasfile: lasio.LASFile) -> dict[str, np.ndarray]:
     :return: Depth data as 'from-to' interval or 'depth' locations.
     """
 
-    if "DEPTH" in lasfile.curves:
-        depths = lasfile["DEPTH"]
-    elif "DEPT" in lasfile.curves:
-        depths = lasfile["DEPT"]
-    else:
-        raise KeyError(
+    depths = None
+    for k in ["DEPTH", "DEPT", "Depth", "Dept"]:
+        if k in lasfile.curves:
+            depths = lasfile[k]
+            break

```

Comment:
how about this:
![image](https://github.com/MiraGeoscience/las-geoh5/assets/81254271/9e0b6a55-42b7-4e02-a2ba-94380d4391e8)
So that we don't mistakely match "dept" in some companies data curve name.

Reply:
Works!